app/views/assignments/_feed_address_form.html.haml

```diff
@@ -19,3 +19,7 @@
         title: 'Click to copy link',
         aria: { label: 'Copy' } }
         %span.far.fa-clipboard
+
+  = link_to roster_assignments_path(@roster, format: :csv), class: 'btn btn-outline-secondary' do
+    CSV Overview
+    %i.fas.fa-download
```

Comment:
yeah, this wraps weird on small devices, but so does the "info circle" part, and addressing it as a whole is not relevant to this PR so I don't really care.